google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/BigtableDataSettings.java

```diff
@@ -463,6 +478,35 @@ public Long getTargetRpcLatencyMsForBatchMutation() {
       return stubSettings.bulkMutateRowsSettings().getTargetRpcLatencyMs();
     }
 
+    /**
+     * Register built in metrics. This is an experimental feature. Please fill up this form to have
+     * your project allow listed for the private preview: https://forms.gle/xuhu6vCunn2MjV2m9
+     */
+    @BetaApi("Built in metric is not currently stable and may change in the future")
+    public Builder registerBuiltinMetrics() throws IOException {
```

Comment:
This is a bit weird, you are registering metrics for all instances, but invoking it on a single instance's settings?

Reply:
I think these need to be static or we should make instrumentation be scoped by instance